utility/routing.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+
+const isEmptyRouting = (content) => {
+    return /\[\s*\]/.test(content);
+};
+
+const getPositionIndex = (content) => {
+    const edgeIndex = content.lastIndexOf(']');
+
+    if(edgeIndex === -1) {
+        console.error('No route found.');
+        process.exit();
+    }
+    return edgeIndex;
+};
+
+const addSeparatorToValue = (insertValue, isEmpty) => {
+    const separator = isEmpty ? '' : ', ';
+
+    return `${separator}${insertValue}`;
+};
+
+const insertToContent = (content, position, insertValue) => {
+    return `${content.slice(0, position)}${insertValue}${content.slice(position, content.length)}`;
+};
+
+const addPageToAppNavigation = (filePath, insertValue) => {
+    const content = fs.readFileSync(filePath).toString();
+    const isEmpty = isEmptyRouting(content);
+    const position = getPositionIndex(content);
+
+    fs.writeFileSync(filePath, insertToContent(content, position, addSeparatorToValue(insertValue, isEmpty)));
+};
+
+
+module.exports = {
```

Comment:
This utility is named `routing` but it works not for any realization of routing and in fact, this is a utility that adds an object to the array in some file. I think it's better to rename it to something more abstract.